app/views/manage/projects/index.html.haml

```diff
@@ -14,8 +14,7 @@
           %th.sortable{ data: { sort: "status asc", reverse: "status desc" } } Status
           %th.sortable{ data: { sort: "updated_at asc", reverse: "updated_at desc" } } Last Updated
           %th Last Contacted
-          %th Location
-          %th.sortable{ data: { sort: "users.last_name asc", reverse: "users.last_name desc" } } Owner
+          %th.sortable{ data: { sort: "users.last_name asc", reverse: "users.last_name desc" } } Project Owner
```

Comment:
This is still sorting on `last_name` but displaying email. Maybe it should change to `users.email` in both of these to the display and sort are consistent.

Reply:
ohhhh 🦅 👀  thank you!